README.md

```diff
@@ -3,7 +3,7 @@
 The open-source platform for monitoring and observability.
 
 [![License](https://img.shields.io/github/license/grafana/grafana)](LICENSE)
-[![Circle CI](https://img.shields.io/circleci/build/gh/grafana/grafana)](https://circleci.com/gh/grafana/grafana)
+[![Drone](https://drone.grafana.net/api/badges/grafana/grafana/status.svg)](https://drone.grafana.net/grafana/grafana/latest?branch=master)
```

Comment:
the url seems has some issue, if you can't get the latest build number easily, with https://drone.grafana.net/grafana/grafana to list all the build on going for grafana should be enough. it is the same as circleci before.

Reply:
![Uploading Screenshot 2021-05-04 at 15.35.13.png…]()